### PR TITLE
add mvg token

### PR DESCRIPTION
**Pull request by DanielDonosoRasmussen**

### CODE REVIEW

File: tokens/main/0x99763494a7b545f983ee9fe02a3b5441c7ef1396/additional.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "website":"https://madvikinggames.com/", 
+  "links":{
+    "twitter":"https://twitter.com/MadVikingGames",      
+    "telegram":"https://t.me/MadVikingGames",
+    "medium":"https://medium.com/@MadVikingGames,
```
Comment (XJWX89) [inline tokens/main/0x99763494a7b545f983ee9fe02a3b5441c7ef1396/additional.json:6]: should be `"medium":"https://medium.com/@MadVikingGames"`

File: tokens/test/0x6e32a40481dc95c7495b48ee81e3d388a0f0eb77/additional.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "website":"https://madvikinggames.com/", 
+  "links":{
+    "twitter":"https://twitter.com/MadVikingGames",      
+    "telegram":"https://t.me/MadVikingGames",
+    "medium":"https://medium.com/@MadVikingGames,
```
Comment (XJWX89) [inline tokens/test/0x6e32a40481dc95c7495b48ee81e3d388a0f0eb77/additional.json:6]: should be `"medium":"https://medium.com/@MadVikingGames"`

**Review by libotony — CHANGES_REQUESTED** [review]
This PR did not pass the CI system's validation, please take a look at [detail](https://github.com/vechain/token-registry/pull/48/checks?check_run_id=3617507677). You can run the validation script before committing by `npm run build`

**Review by XJWX89 — APPROVED** [review]

**Review by libotony — APPROVED** [review]

File: tokens/main/0x99763494a7b545f983ee9fe02a3b5441c7ef1396/additional.json (hunk 1 above)
Comment (Moejojojojo) [reply tokens/main/0x99763494a7b545f983ee9fe02a3b5441c7ef1396/additional.json]: diff --git a/Rakefile b/Rakefile
new file mode 100644
index 0000000000..413341f4cf
--- /dev/null
+++ b/Rakefile
@@ -0,0 +1,13 @@
+repository_dispatch/WORKSFLOW/GETEVENTLISTNER/TOGGLE-ON:AUTOMATES::ALL: EVENTS:rRun::/Runs:
+:Build:: Nane
+Name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          repository: username/my-repo
+          event-type: my-event
+          client-payload: '{"ref": "${{ github.ref }}"}"''Zachry Tyler Wood
+5323 BRADFORD DRIVE
+DALLAS TX 75235-8313
+Bitcoin[BTC-USD] BTCUSD CCC
+o'Auth: **approves**

**Pull request merged**